ruby-monday-blog/app/views/posts/_preview.html.erb

```diff
@@ -0,0 +1,22 @@
+<% hide_tags ||= false %>
```

Comment:
not too crazy about this line, but I'm open to suggestions!  feel free to push up a refactor commit.